translations/de.ts

```diff
@@ -270,7 +251,7 @@ Soll der Proxyserver ignoriert werden und eine Direktverbindung hergestellt werd
         <location filename="../src/widget/form/addfriendform.cpp" line="123"/>
         <source>This Tox ID does not exist</source>
         <comment>DNS error</comment>
-        <translation>Entschuldigung, aber diese ID existiert scheinbar nicht!</translation>
+        <translation>Entschuldigung, aber diese ID existiert scheinbar nicht</translation>
```

Comment:
Personally, I would wait until the source will be changed, so that there is a equality between the different languages. What do you think?